Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
-Resolved, That the preceeding Constitution be laid before the United States in Congress assembled, and that it is the Opinion of this Convention, that it should afterwards be submitted to a Convention of Delegates, chosen in each State by the People thereof, under the Recommendation of its Legislature, for their Assent and Ratification; and that each Convention assenting to, and ratifying the Same, should give Notice thereof to the United States in Congress assembled. Resolved, That it is the Opinion of this Convention, that as soon as the Conventions of nine States shall have ratified this Constitution, the United States in Congress assembled should fix a Day on which Electors should be appointed by the States which shall have ratified the same, and a Day on which the Electors should assemble to vote for the President, and the Time and Place for commencing Proceedings under this Constitution. 
-That after such Publication the Electors should be appointed, and the Senators and Representatives elected: That the Electors should meet on the Day fixed for the Election of the President, and should transmit their Votes certified, signed, sealed and directed, as the Constitution requires, to the Secretary of the United States in Congress assembled, that the Senators and Representatives should convene at the Time and Place assigned; that the Senators should appoint a President of the Senate, for the sole Purpose of receiving, opening and counting the Votes for President; and, that after he shall be chosen, the Congress, together with the President, should, without Delay, proceed to execute this Constitution.
+2: If the exiting State has land borders with a territory which is not a member of the Union, or has water access to the Oceans of the World through waters not claimed as territory by any other State of the Union, the State shall be considered to be sovereign and foreign from these United States on the date one year after the President of these United States has been informed of the State's intent to leave the Union.
 
- 	
-By the unanimous Order of the Convention
+3: Otherwise, the exiting state shall be considered sovereign and foreign on the date two years after the President of these United States has been informed.
 
-W. Jackson  Secretary.
+4: During the period of time between the election of the State to leave the Union and the date of exit, the government of the State shall engage in negotiation of treaties to be activated on the date of exit, to govern trade between the newly separated State and the Union, and to come to agreement regarding the ownership of such lands within the borders of the newly separated State which previously had been purchased for use by the Government of these United States. For the purpose and within the scope of these negotiations, the State shall immediately be considered sovereign and foreign from these United States.
```

Comment:
Observing the negotiations between the EU and the UK for a Brexit deal, it has seemed obvious that the EU still thinks of the UK as a member state, going through a rebellious teenage phase. Let it be clear: the States are sovereign, and the Union has no prior claim upon them.